drools-wb-webapp/pom.xml

```diff
@@ -922,6 +922,24 @@
       <artifactId>drools-scenario-simulation-api</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
@Ginxo Can you please move the dependencies on block which starts in line 1096? Here the are others kogito dependencies which are temporary (probably for the same reason you need to add these new ones?) and planned to be removed with https://issues.jboss.org/browse/KOGITO-459 ? My suggestion is to link all these kogito dependencies in other to be managed (and removed) togheter.

Reply:
@Ginxo Or just put the same comment in this place, your choice :)

Reply:
@yesamer moved, thanks